buildtools/.gitattributes

```diff
@@ -0,0 +1,9 @@
+#
```

Comment:
I do not see the point of this, was added in your branch, what is the purpose of using this?

Reply:
It was auto generated by gradlew when I created the plugin.